Packs/Tripwire/Integrations/Tripwire/Tripwire.py

```diff
@@ -0,0 +1,469 @@
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``
+    :rtype: ``CommandResults``
+    """
+    node_filter = filter_nodes(args)
+    result = client.get_nodes(node_filter)
+    readable_output = tableToMarkdown('Tripwire Nodes list results', result, removeNull=True,
+                                      headers=RULES_HUMAN_READABLE_HEADERS['NODES'])
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Tripwire.Nodes',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def elements_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """tripwire-elements-list command
+
+    :type client: ``Client``
+    :param client: Tripwire client to use
+
+    :type args: `` Dict[str, Any]``
+    :param args:
+        all command arguments, passed from ``demisto.args()``.
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``
+    :rtype: ``CommandResults``
+    """
+    elements_filter = filter_elements(args)
+    result = client.get_elements(elements_filter)
+    readable_output = tableToMarkdown('Tripwire Elements list results', result, removeNull=True,
+                                      headers=RULES_HUMAN_READABLE_HEADERS['ELEMENTS'])
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Tripwire.Elements',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def fetch_incidents(client: Client, max_results: int, last_fetch: str, fetch_filter: str = ''
+                    ) -> Tuple[Dict[str, str], List[dict]]:
+    """
+    :type client: ``Client``
+    :param client: Tripwire client to use
+
+    :type max_results: ``int``
+    :param max_results: Maximum numbers of incidents per fetch
+
+    :type last_fetch: ``str``
+    :param last_fetch:
+        A string contains the last fetched date and time.
+
+    :type fetch_filter: ``Optional[str]``
+    :param fetch_filter:
+        A string that contains the filters for the command.
+
+    :return:
+        A tuple containing two elements:
+            next_run (``Dict[str, str]``): Contains the datetime str that will be
+                    used in ``last_fetch`` on the next fetch.
+            incidents (``List[dict]``): List of incidents that will be created in XSOAR
+
+    :rtype: ``Tuple[Dict[str, int], List[dict]]``
+    """
+
+    incidents: List[Dict[str, Any]] = []
+    last_fetch = datetime.strptime(last_fetch, DATE_FORMAT)
+    last_fetched_ids = demisto.getLastRun().get('fetched_ids', [])
+    alerts = client.get_versions(fetch_filter)
+    alerts = alerts[:int(max_results)]
+    fetched_ids = []
+    for alert in alerts:
+        incident_created_time = datetime.strptime(alert.get('timeDetected'), '%Y-%m-%dT%H:%M:%S.000Z')
+
+        if incident_created_time < last_fetch:
+            continue
+
+        incident_name = alert.get('id')
```

Comment:
let's try to think of a more descriptive incident name than only the ID. maybe something like:
`File <node_id> Changed, latest version ID is <version_id>`